scripts/buildProjects.ps1

```diff
@@ -32,6 +32,10 @@ function BuildServer ( $srcDir, $outDir, $target) {
         $commandArgs += "/p:UseAppHost=false"
     }
 
+    if ($env:RAVEN_BuildOptions -like "*AllowEncryptedOverHttp*") {
```

Comment:
You do not need this. It should be automaticall applied when you set the `$env:RAVEN_BuildOptions` using below csproj change

Reply:
Done